topics/imaging/tutorials/yolo_prediction/tutorial.md

```diff
@@ -0,0 +1,282 @@
+---
+layout: tutorial_hands_on
+title: Object detection and segmentation with YOLO
+
+zenodo_link: ''
+
+questions:
+- How do object detection and segmentation differ in practice?
+- How can I run YOLO models on marine images to detect species using Galaxy?
+- Why does the choice of model type matter?
+
+objectives:
+- Detect marine species in underwater images using a pretrained YOLOv8 model
+- Compare results between detection and segmentation modes
+- Understand the requirements and output types of each model
+
+time_estimation: "45m"
+
+tags:
+- object-detection
+- image-segmentation
+- deep-learning
+- ecology
+
+priority: 5
+
+contributions:
+  authorship:
+    - mjoudy
+    - yvanlebras
+
+follow_up_training:
+  - type: "internal"
+    topic_name: machine-learning
+    tutorials:
+      - ml-advanced-image
+
+---
+
+# Introduction
+
+YOLO (You Only Look Once) is a fast, deep learning-based algorithm for real-time object detection. It predicts object classes and bounding boxes in a single pass over the image. YOLOv8 is a specific version, offering improved accuracy and speed.
+
+In this tutorial, you will use Galaxy to run two types of YOLOv8 models:
+
+1. **Object Detection** using real underwater images from the SEANOE dataset.
+2. **Segmentation** using an Ultralytics demo model on a standard image.
+
+We'll compare both modes and discuss what kind of output they generate and why it matters in bioimage analysis.
+
+---
+
+# Part 1: Detection of marine species
+
+## 🔗 Dataset
+
+We will use selected images from the SEANOE dataset:
+
+👉 [SEANOE 101899 - Deep-sea marine species observations](https://www.seanoe.org/data/00907/101899) (Lebeaud *et al.*, 2024 [https://doi.org/10.17882/101899](https://doi.org/10.17882/101899))
+
+The SEANOE #101899 collection features real underwater images captured by deep‑sea observatories as part of a citizen science initiative called Deep Sea Spy. These non‑destructive imaging stations continuously monitor marine ecosystems and provide snapshots of various fauna. In this dataset, multiple annotators—including trained scientists and enthusiastic citizen scientists—have manually labeled images with polygons, lines, or points highlighting marine organisms. These annotations were then cleaned and converted into bounding boxes to create a training-ready dataset for object detection with YOLOv8. Though the exact species vary, images often include deep-sea fish, species, making this dataset well-suited for practicing detection tasks.
```

Comment:
Showing an example image from SEANOE for visual context would be great along with the explanation which is already written